app/web/platform/nativeLink.ts

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from "react";
+
+export function getReactNativeWebView(): typeof window.ReactNativeWebView {
+  if (
+    typeof window !== "undefined" &&
+    window.ReactNativeWebView !== undefined
+  ) {
+    return window.ReactNativeWebView;
+  }
+}
+
+export function isNativeEmbed(): boolean {
+  return getReactNativeWebView() !== undefined;
```

Comment:
Don't need the undefined unless you just want it to be super explicit.

Reply:
fixed